packages/react-components/react-tag-picker/library/src/components/TagPickerControl/useTagPickerControl.tsx

```diff
@@ -53,7 +54,7 @@ export const useTagPickerControl_unstable = (
   }
 
   const expandIcon = slot.optional(props.expandIcon, {
-    renderByDefault: true,
+    renderByDefault: !noPopover,
```

Comment:
let's have a test for the expand icon too - just JSDOM should be ok for this

Reply:
I added to the cypress test: https://github.com/microsoft/fluentui/pull/32158/files#diff-294ac31394ab40e9246a7dced5c4dcb0e70fa231625bdde6a16c925c0a5bfec8R320

IMHO having verifying on web environment always sounds safer